src/plugins/irc-events/error.js

```diff
@@ -39,7 +39,7 @@ module.exports = function(irc, network) {
 		lobby.pushMessage(client, msg, true);
 
 		if (irc.connection.registered === false) {
-			const random = (data.nick || irc.user.nick) + Math.floor(10 + (Math.random() * 89));
+			const random = (data.nick || irc.user.nick) + Math.floor(Math.random() * 10);
```

Comment:
This is really, [reeeeeally old](https://github.com/thelounge/thelounge/commit/4ef13d6a18551a27d473cb38583bb5300e0cb72c#diff-4f2bea0c158bc3b83f37d4b5c02bb65bR16). It turns out it was generating numbers between 10 (no idea why) and 98 (that's most likely an overlook).

I reduced to 0-9 because this will keep appending numbers anyway, and I find it confusing that requesting `thelounge42` could lead to `thelounge4265`. This change would just make it `thelounge420` to `thelounge429`. If this one already exists, a new integer will be appended, and so on.

Reply:
It generated from `10` so that it always ended up with 2 numbers (if it generated a `5` then it wouldn't have a zero prefix).

Reply:
Oh wow, ok. Well, I definitely prefer the one-int version then!